test/query.test.js

```diff
@@ -3,14 +3,14 @@ const app = require('../src/backend/web/app');
 
 jest.mock('../src/backend/utils/indexer');
 
-describe('Testing query route', () => {
-  test('Testing with valid length of characters', async () => {
+describe('Testing query route with', () => {
```

Comment:
`with` what?  I'm not clear on why you added this.

Reply:
Incorporated in [7dd761b](https://github.com/Seneca-CDOT/telescope/pull/1334/commits/7dd761bc7b9281cd31a572a30424e0c41235cb6b)